### PR TITLE
update to netcoreapp2.1

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
nor sure where in the CI you say what version of netcore is required?

File: src/GraphQL.Tests/GraphQL.Tests.csproj [hunk 1]
```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.0;netcoreapp1.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp1.1;net452</TargetFrameworks>
```
Comment (joemcbride) [inline src/GraphQL.Tests/GraphQL.Tests.csproj]: Instead of replacing `2.0` with `2.1`, I think `2.1` should just be added as another target.

Comment (SimonCropp) [reply src/GraphQL.Tests/GraphQL.Tests.csproj]: My intention here was not to “also test against 2.1”. It was more to get the benefits of build and runtime perf improvements of 2.1 in the context of tests

With that it mind. Do u still want to multi target?

Comment (joemcbride) [reply src/GraphQL.Tests/GraphQL.Tests.csproj]: Yes.  I would prefer to run the tests against all the targeted frameworks.  Call me suspicious but I don't trust that they're 100% compatible.  😀

**Comment by SimonCropp** [comment]
ok changes made. i will rebase my other PRs on this one once it is merged

**Comment by SimonCropp** [comment]
gentle bump

**Pull request merged**